packages/mds-compliance/tests/api.spec.ts

```diff
@@ -1088,4 +1113,33 @@ describe('Tests Compliance API:', () => {
         })
     })
   })
+
+  describe('Verifies max 0 count policy', () => {
+    before('boop', async () => {
```

Comment:
you cannot name it boop

Reply:
I mean, I _CAN_. Though I probably shouldn't 😜